falcon/media/handlers.py

```diff
@@ -39,21 +43,75 @@ def __init__(self, initial=None):
         # Also, this results in self.update(...) being called.
         UserDict.__init__(self, handlers)
 
-    def _resolve_media_type(self, media_type, all_media_types):
-        resolved = None
+    def __setitem__(self, key, item):
```

Comment:
Nitpick: it's more canonical to name `__setitem__` arguments `key` and `value`.

Reply:
True. I guess I wasn't looking to closely. This came from https://github.com/CaselIT/falcon/commit/257a820141961a88ef3ca977a88c30f812da75f4#diff-52784144e74b667a1069b60e8cf6a59f805f05c269e3ce7cccea884167ea5f2fR43

Reply:
Fixed